README.md

```diff
@@ -1,2 +1,9 @@
-# year-cohort-projectName (eg: 22-jul-projectName)
-project template repo
+# FoodDelivery App FST BlueLobsters 2024
+
+FoodDelivery app project set by iO Academy, written in React, using tailwind for styling.
+
+The task was to create a food delivery app using the provided designs. This project was made up of six different stories, each implementing new features and functionality. We decided to go with a mobile first approach, and set out by creating a component tree diagram and drawing a wireframe for the design we were given. We were also provided the API to fetch the relevant data to populate our components with.
```

Comment:
I would specify that this is a frontend app - employers may think you wrote the whole thing.

All this other information about how you're doing it is irrelevant.

Keep: 

- technologies
- that you did it at iO Academy
- scope of the project 
    - frontend
    - 1 week
    - working to provided designs
    - API provided
- that it is a team project

The other information about stories and diagrams and tasks and etc. is just how we always do things so is irrelevant to your target reader (a potential employer).

Reply:
@vincedefreitas will rewrite